src/main/java/com/metabug/persistence/dao/TicketRepository.java

```diff
@@ -4,9 +4,12 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
 import java.util.UUID;
 
 @Repository
 public interface TicketRepository extends JpaRepository<Ticket, UUID> {
     Ticket findByTitle(final String title);
```

Comment:
what if 2 tickets have the same title? Maybe it should be a list?

Reply:
Changed to `findAllByTitle`, but still it's not used anywhere, for now